lib/src/data/simulations.dart

```diff
@@ -63,84 +65,94 @@ class Simulations with ChangeNotifier {
       ),
       SimulationCard(
         id: 1,
+        simulationName: 'Selection Sort',
```

Comment:
Here too. Move it to the end and restore the original ids of the existing simulations.